src/components/Accordion/Accordion.stories.tsx

```diff
@@ -110,6 +113,206 @@ export const WithDifferentAccordionItems: Story<AccordionProps> = () => {
     );
 };
 
+const customHeader: FC<AccordionHeaderProps> = ({ isOpen, children }) => (
+    <div
+        className={merge([
+            "tw-px-6 tw-py-4 tw-bg-black-5 tw-flex tw-justify-between tw-font-bold tw-items-center",
+            isOpen ? "tw-text-black" : "tw-text-black-60",
+        ])}
+    >
+        {children}
+        <AccordionHeaderIcon isOpen={isOpen} type={FieldsetHeaderType.Accordion} size={AccordionHeaderIconSize.Small} />
+    </div>
+);
+
+const Code: FC = ({ children }) => (
+    <code className="tw-bg-black-5 tw-rounded tw-px-2 tw-text-box-negative-strong tw-text-s">{children}</code>
+);
+
+const PropsTable: FC<{ rows: [ReactNode, ReactNode, ReactNode][] }> = ({ rows }) => (
```

Comment:
Please add the [storybook controls](https://storybook.js.org/docs/react/essentials/controls) so we have props documentation for free and a tool to play around :)

Reply:
Nice, yes I get it, it's also not really able to handle components via props.